.gitignore

```diff
@@ -14,3 +14,5 @@ profile.out
 /dist
 *.test
 tests/__snapshot__
+/vendor/github.com/loft-sh/vcluster-pro/
+/vendor/github.com/FabianKramm/private-test/
```

Comment:
Do we need this, or is it a leftover development snippet?

Reply:
Yeah we don't need that, good catch